Made simple header with dynamic breadcrumbs

luuk-royal opened the pull request.

luuk-royal committed:
Made simple header with dynamic breadcrumbs

luuk-royal committed:
header and pageLayout cleanup

luuk-royal committed:
header and organisation page cleanup

VincentVel commented:
hydration error in console.
styling van header is niet volgens de mockup.
styling van tabel is niet volgens de mockup.
create event button is nog steeds genaamd "create organisation".
tabel header staat event terwijl dit de name is van de event.
ID laten zien in tabel?
Je kunt klikken op het id van de organisatie binnen de breadcrumb dit brengt je naar een 404.
knop van create event in Dialog is nog steeds "create organisation".
dialog wordt niet gesloten op het moment van aanmaken van een event.
bij het sluiten van de dialog wordt de waarde van de input niet gereset.
Bij het aanmaken van meer een stuk of 9 wordt het scherm scrollable.
Bij het verwijderen van een event wordt het Id als undefined verstuur waardoor deze niet wordt verwijderd.
bij het ophalen van de events op organisatie na het proberen te verwijderen van een event zal er een request worden gedaan met organisationId als undefined.
Bij het updaten zijn roganisationId en eventId undefined
Er zijn nog errors in de code, deze zijn gericht op Typescript errors

luuk-royal committed:
Fixed typing and typescript issues.
Name is now properly reset after event is created.
Fixed some copy pasta issues.
Fixed handleUpdate of events not being put and sending the event.
Added in the organisationId to onChange so the data is always fetched correctly after changes.
Fixed request not getting the id's and getting undefined instead.
Changed wildcard typing to regular imports as typescript gives warnings about wildcards.
Fixed id's being clickable and routing back to a 404 page, ID's in the breadcrumb can no longer be clicked.

luuk-royal committed:
added in styling for Header

luuk-royal committed:
updated dynamicBreadcrumb, id's are now replaced by their respective objects name
